src/components/Main.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+import { getAllArticles } from "../../api";
+
+function Main() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((data) => {
+      setArticles(data);
+    });
+  }, []);
+
```

Comment:
Please add a loading state and keep adding them throughout the project when content is being loaded from your API. Otherwise the page is blank and the user doesn't know articles are on the way. This is particularly important because the first time your API is woken up it takes a long time. Here are some notes on loading states if you're not sure 🙂 
https://notes.northcoders.com/courses/js-front-end/useEffect#loading-patterns